components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -257,6 +257,9 @@ func Run(options ...RunOption) {
 		go analyseConfigChanges(ctx, cfg, analytics, gitpodConfigService, gitpodService)
 	}
 
+	topService := NewTopService(Top)
```

Comment:
(nit) I think for usual clients it should be just `NewTopService`

Reply:
@akosyakov do you mean without parameters?